src/games/gameitem.py

```diff
@@ -204,6 +196,14 @@ def update(self, message, client):
             client.usersUpdated.emit(self.players)
             return
 
+        # Map preview code
```

Comment:
I'm not over the 'if not hidden' part, I plan to put it back in.
(and I don't understand the 'less verbose' part.)